rudder_airflow_provider/operators/rudderstack.py

```diff
@@ -87,6 +87,7 @@ def __init__(
         request_max_retries: int = DEFAULT_REQUEST_MAX_RETRIES,
         poll_timeout: float = None,
         poll_interval: float = DEFAULT_POLL_INTERVAL_SECONDS,
+        parameters: Optional[List[str]] = None,
```

Comment:
some documentation about it in `README.md` would be nice

Reply:
@atzoum that should now be good